app/src/main/java/com/example/directory/repository/UsersRepository.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.directory.repository
+
+import com.example.directory.api.UserService
+import com.example.directory.models.User
+import io.reactivex.Observable
+
+class UsersRepository(private val userService: UserService) {
```

Comment:
Lets rename the class to UserApi and create a UserService which will be used by viewmodel to call the api.
UserService can have a lot of business logic that can be made before making the exact api call